mosviz/controls/saved_slits.yaml

```diff
@@ -0,0 +1,44 @@
+nirspec_msa:
+  name: "NIRSpec MSA (Single Slit)"
+  length: 3.3
+  width: 0.2
```

Comment:
@robelgeda Silly question and we might have talked about this, but are units relevant here?  Should unit information be stored so it is super clear what the units are or is it obvious enough to the user?

Reply:
This is a good idea I will add that

Reply:
To make this a bit clearer, what about something like:
```
  length: 3.3 arcsec
  width: 0.2 arcsec
```
instead of what's now here...?  You'll have to pass the values through the `Quantity` initializer, but ``Quantity("3.3 arcsec")`` works out of the box so this should be pretty easy for users to understand...